singer_sdk/sinks/sql.py

```diff
@@ -82,11 +82,31 @@ def schema_name(self) -> Optional[str]:
         Returns:
             The target schema name.
         """
+        # Get the current SQL Dialect being used
+        target_sqla_dialect = self.connection.engine.dialect.name
```

Comment:
Thanks, that is very helpful to know.  Since I am moving it from the SDK `SQLSink` to `MSSQLSink` I was going to drop the dialect check since I know it has been set to `mssql` or would you recommend leaving it in place.

Reply:
@BuzzCutNorman yeah it makes sense to just remove this line in the SDK 👍

Reply:
@edgarrmondragon the line `target_sqla_dialect = self.connection.engine.dialect.name` has been removed